exercises/custom-converter/README.md

```diff
@@ -0,0 +1,134 @@
+# Exercise 1: Implement a Custom Data Converter
+
+During this exercise, you will: 
+
+* Output typical payloads from a Temporal Workflow using the default Data Converter
+* Implement a Custom Data Converter that encrypts Workflow output
+* Implement a Failure Converter and demonstrate parsing its output
+
+Make your changes to the code in the `practice` subdirectory (look for 
+`TODO` comments that will guide you to where you should make changes to 
+the code). If you need a hint or want to verify your changes, look at 
+the complete version in the `solution` subdirectory.
+
+
+## Part A: Implement a Custom Data Converter
+
+1. Defining a Custom Data Converter is a straightforward change to your existing
+   Worker and Starter code. The example in the `practice` subdirectory of this
+   exercise is missing the necessary change to use a Custom Data Converter,
+   meaning that you can run it out of the box, and produce JSON output using the
+   Default Data Converter. You'll do this first, so you have an idea of the
+   expected output. First, start the Worker:
+
+   ```shell
+   go run worker/main.go
+   ```
+
+2. Next, run the Workflow starter:
+
+   ```shell
+   go run starter/main.go
+   ```
+
+3. After that, you can use the `temporal` CLI to show the Workflow result:
+
+   ```shell
+   temporal workflow show -w converters_workflowID
+   ```
+
+   ```
+   Progress:
+     ID          Time                     Type
+      1  2024-01-16T17:10:53Z  WorkflowExecutionStarted
+      2  2024-01-16T17:10:53Z  WorkflowTaskScheduled
+      3  2024-01-16T17:10:53Z  WorkflowTaskStarted
+      4  2024-01-16T17:10:53Z  WorkflowTaskCompleted
+      5  2024-01-16T17:10:53Z  MarkerRecorded
+      6  2024-01-16T17:10:53Z  MarkerRecorded
+      7  2024-01-16T17:10:53Z  ActivityTaskScheduled
+      8  2024-01-16T17:10:53Z  ActivityTaskStarted
+      9  2024-01-16T17:10:53Z  ActivityTaskCompleted
+     10  2024-01-16T17:10:53Z  WorkflowTaskScheduled
+     11  2024-01-16T17:10:53Z  WorkflowTaskStarted
+     12  2024-01-16T17:10:53Z  WorkflowTaskCompleted
+     13  2024-01-16T17:10:53Z  WorkflowExecutionCompleted
+
+   Result:
+     Status: COMPLETED
+     Output: ["Received Plain text input"]
+   ```
+
+   You should now have an idea of how this Workflow runs ordinarily — it outputs
+   the string "Received Plain text input". In the next step, you'll add a Custom
+   Data Converter.
+4. To add a Custom Data Converter, you don't need to change anything in your
```

Comment:
Fair point -- I've inverted the order of what you don't need to do and what you do need to do, and clarified that you should go ahead and make the change, so that it's less ambiguous but still reinforces the parts I want to reinforce.